### PR TITLE
M2 3058

### PR DESCRIPTION
**Pull request by ksuh90**

### CODE REVIEW

File: jerry/commands/send_daily_sales_email.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    Implements 'jerry send-daily-sales-email'.
+    """
+
+    def __init__(self):
+        super(SendDailySalesEmail, self).__init__('Send daily sales report email')
+        self.jinja2_env = Environment(loader=PackageLoader(_package, 'templates'),
+                                      autoescape=select_autoescape(['html']))
+
+        self.parser.add_argument('-r', '--recipient',
+                                 help='Ricipient email(s)',
```
Comment (calebgrana) [inline jerry/commands/send_daily_sales_email.py]: Minor typo here

File: jerry/commands/send_daily_sales_email.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    def run(self):
+        super(SendDailySalesEmail, self).run()
+
+        tableau_helper = TableauHelper(TABLEAU_CONFIG['user'],
+                                       TABLEAU_CONFIG['password'],
+                                       TABLEAU_CONFIG['site_id'],
+                                       TABLEAU_CONFIG['server_address'])
+
+        # Today's date
+        date = datetime.date.today().strftime('%d, %b %Y')
```
Comment (calebgrana) [inline jerry/commands/send_daily_sales_email.py]: Maybe get yesterday's date?

File: jerry/commands/send_daily_sales_email.py (hunk 1 above)
Comment (ksuh90) [reply jerry/commands/send_daily_sales_email.py]: Fixed

File: jerry/commands/send_daily_sales_email.py (hunk 2 above)
Comment (ksuh90) [reply jerry/commands/send_daily_sales_email.py]: Fixed

**Pull request merged**